ambry-protocol/src/main/java/com/github/ambry/protocol/PutRequest.java

```diff
@@ -213,6 +268,9 @@ protected void prepareBuffer() {
     if (keyLength > 0) {
       bufferToSend.writeBytes(blobEncryptionKey);
     }
+    if (versionId >= PUT_REQUEST_VERSION_V5) {
+      bufferToSend.writeByte(isCompressed ? 1 : 0);
```

Comment:
```
    if (versionId >= PUT_REQUEST_VERSION_V5) {
      bufferToSend.writeByte(Boolean.compare(isCompressed, true));
    }
```

Reply:
Isn't it easier to read using (isCompressed ? 1 : 0)? This pattern is used in the rest of the Ambry code too.

Reply:
Both are easy enough but if Java provides a method, I think it's better to use it. 
Agreed this pattern is in rest of Ambry, but we can try changing that henceforth.

Reply:
But nvm, the pr lgtm.